sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLQueryPlanner.java

```diff
@@ -81,6 +89,13 @@
   "nullness" // TODO(https://issues.apache.org/jira/browse/BEAM-10402)
 })
 public class ZetaSQLQueryPlanner implements QueryPlanner {
+  public enum Calc {
```

Comment:
Nit: can we rename this to avoid confusion with the actual [Calc](https://github.com/apache/calcite/blob/master/core/src/main/java/org/apache/calcite/rel/core/Calc.java)?

Reply:
How about `CalcRuleEnum`?

Reply:
On second thought, why do we have the enum at all? Can we not pass the rules directly?

Reply:
I was trying not to add the ability to select arbitrary rules to the public API, but I'll do that instead.

Reply:
> I was trying not to add the ability to select arbitrary rules to the public API, but I'll do that instead.

Good point. I'll leave it up to you, I'm fine with either way.

Reply:
I like your suggestion for now, simpler is better.